commit 12fb28ea6b1cf24bde27c406eb38ee3c108d73f9
Author: Ben Cheatham <Benjamin.Cheatham@amd.com>
Date:   Mon Mar 11 09:25:06 2024 -0500

    EINJ: Add CXL error type support
    
    Move CXL protocol error types from einj.c (now einj-core.c) to einj-cxl.c.
    einj-cxl.c implements the necessary handling for CXL protocol error
    injection and exposes an API for the CXL core to use said functionality,
    while also allowing the EINJ module to be built without CXL support.
    Because CXL error types targeting CXL 1.0/1.1 ports require special
    handling, only allow them to be injected through the new cxl debugfs
    interface (next commit) and return an error when attempting to inject
    through the legacy interface.
    
    Reviewed-by: Jonathan Cameron <Jonathan.Cameron@huawei.com>
    Signed-off-by: Ben Cheatham <Benjamin.Cheatham@amd.com>
    Link: https://lore.kernel.org/r/20240311142508.31717-3-Benjamin.Cheatham@amd.com
    Signed-off-by: Dan Williams <dan.j.williams@intel.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 2ecaaec6a6bf..90cf8403dd17 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -5289,6 +5289,7 @@ M:	Dan Williams <dan.j.williams@intel.com>
 L:	linux-cxl@vger.kernel.org
 S:	Maintained
 F:	drivers/cxl/
+F:	include/linux/cxl-einj.h
 F:	include/linux/cxl-event.h
 F:	include/uapi/linux/cxl_mem.h
 F:	tools/testing/cxl/